flexibleengine/config_test.go

```diff
@@ -534,3 +535,27 @@ func TestAccServiceEndpoints_Others(t *testing.T) {
 	actualURL = serviceClient.ResourceBaseURL()
 	testCheckServiceURL(t, expectedURL, actualURL, "RTS")
 }
+
+func TestCheckOssEndpoint(t *testing.T) {
+	cfg := &Config{
+		Region: "eu-west-0",
+		Cloud:  "prod-cloud-ocb.orange-business.com",
+	}
+
+	// without customizing OSS endpoint in Config
+	expected := "https://oss.eu-west-1.prod-cloud-ocb.orange-business.com/"
+	th.AssertEquals(t, expected, getOssEndpoint(cfg, "eu-west-1"))
+
+	// with customizing OSS endpoint in Config
+	cfg.Endpoints = map[string]string{
+		"obs": "https://oss.eu-west-0.prod-cloud-ocb.orange-business.com/",
```

Comment:
The key of `getOssEndpoint` is **oss**, but we add **obs** in customizing endpoints.

It looks strange that for Object Storage Service user should add two keys if they want to customize the endpoint.
- **oss** for s3 resources
- **obs** for obs resources

I will check the short name of Object Storage Service and combine them into one.

Reply:
You're right, I see that this error existing before, and oss key doesn't exists in customizing endpoints.

Reply:
**oss** used to be the short name of Object Storage Service, now the it's called **OBS**
but the endpoint is still using `oss.eu-west-0.prod-cloud-ocb.orange-business.com`

so, we should get the **obs** key in `getOssEndpoint` function.

Reply:
This is correct now